src/scripts/main.js

```diff
@@ -1,3 +1,5 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.cssText = 'top: 50%; left: 50%; transform: translate(-50%, -50%);';
```

Comment:
It's a tricky solution, but you should use js properties from the lesson to practice them

Reply:
but its properties from lesson lol